radish34/api/src/resolvers/msa.js

```diff
@@ -94,7 +93,7 @@ export default {
         constants: {
           zkpPublicKeyOfBuyer: zkpPublicKey,
           zkpPublicKeyOfSupplier: _supplier.zkpPublicKey,
-          ...args.input,
+          ...input,
```

Comment:
Does `input` get assigned to anything? Seems like this should stay as `args.input`?